.github/workflows/codeql-analysis.yml

```diff
@@ -2,9 +2,9 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, main ]
```

Comment:
curious -- why do we want to keep master here? Longer term I think master would be deleted and only merging to main would be allowed so maybe we should start doing it from now?